documentation/src/main/markdown/packagemgrs/opam.md

```diff
@@ -31,6 +32,9 @@ to get the list of resolved packages installed in the current [switch](https://o
 
 ## OPAM Lock Detector
 
-The OPAM Lock Detector does not find transitives for the project and is considered a LOW accuracy Detector. OPAM Lock Detector will run if HIGH accuracy Detectors cannot, and the project contains `<pkgname>.opam.locked` and `<pkgname>.opam` files in the top level directory.
+The OPAM Lock Detector is considered a LOW accuracy Detector. OPAM Lock Detector will run if HIGH accuracy Detectors cannot, and the project contains `<pkgname>.opam.locked` and `<pkgname>.opam` files in the top level directory.
 
-OPAM Lock Detector will parse both `<pkgname>.opam.` and `<pkgname>.opam.locked` files to gather the list of direct project dependencies.
\ No newline at end of file
+OPAM Lock Detector will parse both `<pkgname>.opam` and `<pkgname>.opam.locked` files to gather the list of dependencies.
+
+OPAM Lock Detector will declare a dependency as direct if the dependency is present in both `<pkgname>.opam` and `<pkgname>.opam.locked` file. Otherwise, the dependency will be deemed as transitive.
+Based on the information available, [detect_product_short] cannot determine the position of the transitive dependency in the graph, and will note the dependency under a placeholder "component" named *Additional_Components*.
```

Comment:
If I'm understanding correctly, there will be some arbitrary parent component created to house all the transitive dependencies. 

If that is the case maybe we could word this better, something along the lines of: 
Based on the information available, because [detect_product_short] cannot determine the position of a transitive dependency in the graph, it will note the dependency under a placeholder "parent component" named *Additional_Components*.